AaronRichardsWFS/Wesley-Central-Access-master
# edit client information when creating inquiry

Able to edit the client information via a collapsible form while creating a new inquiry for that client
How to test:
- go to the create new inquiry for an existing client
- change some information for the client
- create the inquiry
- check if the client information is updated
